backend/src/main/java/com/redhat/cloud/notifications/db/ApplicationResources.java

```diff
@@ -65,10 +65,18 @@
     }
 
     public Uni<List<Application>> getApplications(String bundleName) {
-        String query = "FROM Application WHERE bundle.name = :bundleName";
-        return session.createQuery(query, Application.class)
-                .setParameter("bundleName", bundleName)
-                .getResultList();
+        String sql = "FROM Application";
+        if (bundleName != null) {
+            sql += " WHERE bundle.name = :bundleName";
+        }
+
+        var query = session.createQuery(sql, Application.class);
```

Comment:
```suggestion
        let query = session.createQuery(sql, Application.class);
```

Reply:
![Just kidding](https://ultimastatus.com/wp-content/uploads/2021/06/hahaha-just-kidding-meme.jpg)

Reply:
You got me there :grin: